Tests/Shared.Specs/ExceptionAssertionSpecs.cs

```diff
@@ -895,6 +894,62 @@ public void When_no_exception_should_be_thrown_and_none_was_it_should_not_throw(
             //-----------------------------------------------------------------------------------------------------------
             foo.Invoking(f => f.Do()).Should().NotThrow();
         }
+
+        private static void ThrowUntil(Stopwatch watch, int throwTime)
```

Comment:
🔧 Take two `TimeSpan` parameters instead and pass in `StopWatch.Elapsed` as the first argument.

Reply:
🤔 I prefer not to use helper methods in test. They tend to obscure the test.